sensirion_common.h

```diff
@@ -86,6 +86,13 @@ uint8_t sensirion_common_generate_crc(uint8_t *data, uint16_t count);
 int8_t sensirion_common_check_crc(uint8_t *data, uint16_t count,
                                   uint8_t checksum);
 
+/**
+ * Send a general call reset.
```

Comment:
Add a warning: "This will reset all devices on the i2c bus"

Reply:
Fixed.